packages/build/src/config/config.ts

```diff
@@ -46,4 +46,5 @@ export interface Config {
   artifactUrlExtraTag?: string;
   manpage?: ManPageConfig;
   isDryRun?: boolean;
+  isAuxiliaryOnly?: boolean;
```

Comment:
I'd generally consider renaming this to something that's more clearly scoped to publishing tasks, e.g. `isAuxiliaryPackagesOnly`, since the current name on its own doesn't carry a lot of meaning